Makefile

```diff
@@ -161,7 +161,7 @@ endif
 build: build-image
 	@echo "+ Building bin/$(ARCH)/nginx-ingress-controller"
 	@$(DEF_VARS) \
-	GOBUILD_FLAGS=$(GOBUILD_FLAGS) \
+	GOBUILD_FLAGS="$(GOBUILD_FLAGS)" \
```

Comment:
Quotes needed to avoid `/bin/sh: -a: command not found` when passing custom flags.